src/io/flutter/sdk/FlutterSdkUtil.java

```diff
@@ -35,13 +36,25 @@
 
 
 public class FlutterSdkUtil {
+  /** The environment variable to use to tell the flutter tool which app is driving it. */
+  public static final String FLUTTER_ENVIRONMENT = "FLUTTER_ENVIRONMENT";
```

Comment:
Perhaps `FLUTTER_HOST_ENV` then? The var could represent IntelliJ, or it could represent Travis.